packages/artifacts/src/download/cdns.ts

```diff
@@ -0,0 +1,12 @@
+import type { Project } from '../projects'
+import type { Strings, Version } from './types'
+
+const CDNS = ['https://unpkg.com', 'https://raw.githubusercontent.com', 'https://cdn.jsdelivr.net'] as const
+
+export type Urls = Strings<typeof CDNS>
```

Comment:
stricter dynamic type based on the length of `CDNS`:
`[string, string, string]` instead `string[]`